test/public/logs/overview.test.js

```diff
@@ -107,7 +110,8 @@ module.exports = () => {
 
         // Insert some other text into the filter
         await page.type('#titleFilterText', ' bogusbogusbogus');
-        await waitForTimeout(300);
+        // Wait until there is no row with data left in the table
+        await page.waitForFunction(() => document.querySelector('#row1') === null, { timeout: 1500 });
```

Comment:
It's simpler to use directly waitForSelector in those cases
```suggestion
        await page.waitForSelector('#row1', {timeout: 1500, hidden: true});
```

Reply:
Done, for these types of checks I used extracted function waitForTableToLength using expectedSize 0, this checks empty row and loading row as well